Makefile

```diff
@@ -0,0 +1,38 @@
+export V?=0
+
+CUR_PATH := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
```

Comment:
Do you really need this? I don't think so.

Reply:
@jforissier you mean that this way of getting current path is a bit weird?

Reply:
`$(CURDIR)` is builtin so you should use it instead, but here I don't think you need a path at all. What about replacing all occurrences of `$(CUR_PATH)/` with nothing?

Reply:
I decided to use this way because `$(CURDIR)` doesn't guarantee to store a path, where `Makefile ` is stored (explained here https://stackoverflow.com/a/22214042)

Reply:
You're trying to address problems that you don't have. Tell me why simply removing `$(CUR_PATH)` does not work.